package.json

```diff
@@ -21,6 +21,7 @@
   },
   "homepage": "https://github.com/LLK/scratch-audio#readme",
   "devDependencies": {
+    "audio-context": "^1.0.1",
```

Comment:
Since this runs as part of the application, we should pin it to the exact version `1.0.1` rather than the major version `^1.0.1`.